frontend/common/src/components/Accordion/Accordion.docs.mdx

```diff
@@ -8,17 +8,17 @@ This is a styled version of [@radix-ui/accordion](https://www.radix-ui.com/docs/
 
 ## Subcomponents
 
- - `Accordion.Root`
```

Comment:
Happy to see some formatting fixes but do they belong in this PR? 😛

Reply:
it's a personal thing, sometimes people have commits with changes mixed with Prettier formatting making things nigh-unintelligible 
just getting ahead of that, it was just running prettier and formatting schema so not even a minute to do 😃

Reply:
I agree with Eric. Let's not yet forget the lessons of the hydrogen PR. :laughing:

Reply:
I agree with Eric.  Let's not quickly forget the lessons of the h2 PR.  :laughing:  I'm OK with tidying files you're working on but if it has nothing to do with your PR then let's leave them alone.